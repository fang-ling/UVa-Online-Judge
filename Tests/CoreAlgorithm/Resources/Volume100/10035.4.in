1 10000
158 695
145875 55
25961 45686
5 58214
666658 554215
584877385 591338400
336295784 639583897
200306876 736951791
649601847 96403807
689942810 312242372
171746653 733051968
5475 541555
65485 4556
0 0
