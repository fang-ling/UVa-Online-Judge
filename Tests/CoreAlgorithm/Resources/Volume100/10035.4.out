No carry operation.
2 carry operations.
2 carry operations.
3 carry operations.
No carry operation.
4 carry operations.
5 carry operations.
5 carry operations.
3 carry operations.
5 carry operations.
5 carry operations.
4 carry operations.
3 carry operations.
4 carry operations.
